tests/ai/test_databricks_client.py

```diff
@@ -30,55 +26,25 @@
     DatabricksFunctionClient,
     extract_function_name,
 )
-
-CATALOG = "ml"
-SCHEMA = "serena_uc_test"
-
-_logger = logging.getLogger(__name__)
-
-
-# TODO: CI -- two test cases
-# 1. python 3.10 with databricks-connect 15.1.0, no cluster_id, use serverless
-# 2. python 3.9 with databricks-sdk, with cluster_id
-@pytest.fixture
-def client() -> DatabricksFunctionClient:
-    with mock.patch(
-        "unitycatalog.ai.databricks.get_default_databricks_workspace_client",
-        return_value=mock.Mock(),
-    ):
-        return DatabricksFunctionClient(warehouse_id="warehouse_id", cluster_id="cluster_id")
-
-
-@pytest.fixture
-def serverless_client() -> DatabricksFunctionClient:
-    return DatabricksFunctionClient()
-
-
-def random_func_name():
-    return f"test_{uuid.uuid4().hex[:4]}"
-
-
-@contextmanager
-def generate_func_name_and_cleanup(client: DatabricksFunctionClient):
-    func_name = random_func_name()
-    try:
-        yield func_name
-    finally:
-        try:
-            client.client.functions.delete(f"{CATALOG}.{SCHEMA}.{func_name}")
-        except Exception as e:
-            _logger.warning(f"Fail to delete function: {e}")
+from unitycatalog.test_utils import (
+    CATALOG,
+    SCHEMA,
+    client,  # noqa: F401
```

Comment:
why do we need F401?

Reply:
Because it's used as fixture in the test but ruff complained 
<img width="705" alt="image" src="https://github.com/user-attachments/assets/33d638ff-c8f6-4511-bad1-9db09e7692f8">